api/src/helpers/transferCredits.ts

```diff
@@ -17,10 +19,11 @@ type TransferredMiscRow = {
   units: number | null;
 };
 
-type TransferredMiscSelectedRow = {
+type GroupedUncategorizedTransfer = {
```

Comment:
Consider adding a docstring to this type because "grouped" is not inherently clear on its meaning